files/scripts/supervisor-proc-exit-listener

```diff
@@ -106,6 +108,13 @@ def get_autorestart_state(container_name):
 
     return is_auto_restart
 
+def publish_events(process_name, container_name):
+    events_handle = swsscommon.events_init_publisher(EVENTS_PUBLISHER_SOURCE)
```

Comment:
Again, this would block for async connect.

Reply:
Done